boskos/janitor/deployment.yaml

```diff
@@ -61,4 +61,24 @@ spec:
       - name: service
         secret:
           secretName: service-account
-
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: boskos-janitor-aws
+  labels:
+    app: boskos-janitor-aws
+  namespace: test-pods
+spec:
+  replicas: 4  # 4 distributed janitor instances
+  template:
+    metadata:
+      labels:
+        app: boskos-janitor-aws
+    spec:
+      terminationGracePeriodSeconds: 300
+      containers:
+      - name: boskos-janitor-aws
+        image: gcr.io/k8s-testimages/janitor-aws:latest
```

Comment:
@krzyzacy I used `latest` for the tag here, since I cannot predict the image tag that will be generated when you build.

Reply:
can you add a TODO? I'll update the tag

Reply:
Done!